packages/sdk/.env

```diff
@@ -0,0 +1,3 @@
+# ensure these are added in the key/values in the rollup.config.js file
+API_URL=https://api.us.flatfile.io
+MOUNT_URL=https://app.flatfile.io
```

Comment:
Is `.env` intentionally checked into repo?

Reply:
it was already checked in previous (at the root of project), and they're used in the code for sdk. webpack just replaces them with these values.

we can hide it if needed! I figured they weren't sensitive, but it does feel weird to show a `.env` - totally agree! :D